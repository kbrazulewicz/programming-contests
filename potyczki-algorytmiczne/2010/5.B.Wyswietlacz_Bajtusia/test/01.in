5
15 10 1572389410
3 5 10000
10 6 205030
7 11 10726341515
1 5 82345
