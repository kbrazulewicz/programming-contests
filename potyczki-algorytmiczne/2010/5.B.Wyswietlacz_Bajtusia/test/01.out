9999999977
98100
999998
99999762111
98542
